crypto/ex_data.c

```diff
@@ -63,8 +63,9 @@ static void cleanup_cb(EX_CALLBACK *funcs)
 void crypto_cleanup_all_ex_data_int(OPENSSL_CTX *ctx)
 {
     int i;
-    OSSL_EX_DATA_GLOBAL *global = openssl_ctx_get_ex_data_global(ctx);
+    OSSL_EX_DATA_GLOBAL *global;
```

Comment:
again, why split?  please don't make needless changes if the code already follows the style.

Reply:
See the comment above.

If you want I can put as was before, but (again). Put this in the coding guideline to keep it consistent.